Source/WebInspectorUI/UserInterface/Views/DOMTreeElement.js

```diff
@@ -1057,7 +1057,7 @@ WI.DOMTreeElement = class DOMTreeElement extends WI.TreeElement
         function keyupListener(event)
         {
             if (closingTagElement)
-                closingTagElement.textContent = "</" + tagNameElement.textContent + ">";
+                closingTagElement.textContent = "</" + tagNameElement.textContent.trim().split(" ")[0] + ">";
```

Comment:
rather than split the entire string, i think it'd be more efficient to do something like this
```js
if (closingTagElement) {
    let textContent = tagNameElement.textContent.trimStart();
    let whitespaceIndex = textContent.search(/\s/);
    if (whitespaceIndex === -1)
        textContent = textContent.slice(0, whitespaceIndex);
    closingTagElement.textContent = "</" + textContent + ">";
```

Reply:
Neat! I forgot about `trimStart()`. But in this case `trim()` is still suitable to remove any stray spaces before and after a single new string.